jaypeasee/national-parkfinder
# Refactor- Interface Structure

## Is this a fix or a feature?
Fix

## What is the change?
Moved (almost) all interfaces into a global `interfaces.tsx` file and cleaned up unnecessary interface declarations. The `interfaces.tsx` file in the `ParkContainer` folder no longer exists. There was one interface for `choosePark` which was a function that I was unable to move because it holds a parameter that is relevant to the files themselves.

## What does it fix?
Ultimately, this just tidies things up a bit.

## Where should the reviewer start?
`interfaces.tsx` from `src` folder

## How should this be tested?
Run the application to make sure data still displays without errors.

@saraho1123
@jaypeasee
@baileydunning
